support-analytics/terraform/main.tf

```diff
@@ -12,3 +12,16 @@ resource "azurerm_resource_group" "resource-group" {
   location = var.location
   tags     = local.common-tags
 }
+
+resource "azurerm_monitor_action_group" "critical-alerts-action" {
+  name                = "CriticalAlertsAction"
+  resource_group_name = azurerm_resource_group.resource-group.name
+  short_name          = "p0action"
+
+  email_receiver {
+    name                    = "sendtoadmin"
+    email_address           = "Jonathan.ABBOTT@EDUCATION.GOV.UK"
```

Comment:
I appreciate this is not likely to be a long-standing addition, but is it worth moving this to a `list(string)` variable?

Reply:
Thought about a list of emails, however my thinking was we should use a mail group and avoid managing multiple emails, etc.

What do you think?